CWP/papers/HSF-CWP-2017-01_roadmap/latex/cwp.bib

```diff
@@ -2372,3 +2371,13 @@ @article{1742-6596-664-8-082046
   url={http://stacks.iop.org/1742-6596/664/i=8/a=082046},
   year={2015},
 }
+
+@online{Spark,
```

Comment:
I guess the web site is incorrect below (I think the paper I added should also be referenced)

Reply:
From looking at your PR, yes, your paper is a good citation to have. But why do you say https://www.scala-lang.org/ is wrong?

Reply:
Oh, I managed to do a wrong copy-and-paste somehow! Thanks for spotting this.

Reply:
I will get this fixed after lunch.